samples/DgraphJavaSample/build.gradle

```diff
@@ -17,10 +17,13 @@ repositories {
 
 dependencies {
  	// Use Dgraph Java client
-    compile 'io.dgraph:dgraph4j:21.12.0'
+    implementation 'io.dgraph:dgraph4j:21.12.0'
 
     // Use JUnit test framework
-    testCompile 'junit:junit:4.12'
+    runtimeOnly 'junit:junit:4.12'
```

Comment:
there is no concept of test dependency any more?

Reply:
This is a good catch...  there is, `testImplementation`, I modified the build.Gradle files